### PR TITLE
Added Jest v26.6.3, creates test GitHub Action

### PR DESCRIPTION
**Pull request by Bwoltz**
Added Jest. There aren't any tests yet, I will add them once I fix my previous pull request and have it merged.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,10 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
-    "eslint": "^7.24.0"
+    "eslint": "^7.24.0",
+    "jest": "^26.6.3"
+  },
+  "scripts": {
+    "test": "jest --verbose ./test"
   }
```
Comment (mattxwang) [inline package.json:32]: I think this is duplicated with the `scripts` key above - let's just change the existing `test` to now use jest!

**Review by mattxwang — CHANGES_REQUESTED** [review]
Overall looks good to me, left a tiny note about something gone wrong in the `package.json`.

Either in this PR, or in an immediate follow-up, we should add a GitHub Action that fires off our tests on every commit. It would be a pretty common action; [the website config is very similar](https://github.com/uclaacm/website/blob/master/.github/workflows/node-build.yml), and we'd just remove the build step and uncomment the tests step.

File: package.json (hunk 1 above)
Comment (Bwoltz) [reply package.json:32]: Oh yep, didn't see it before. Thanks.

File: .github/workflows/npm-test.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
```
Comment (mattxwang) [inline .github/workflows/npm-test.yml:10]: Let's rename the job:

```suggestion
  test:
```

**Review by mattxwang — APPROVED** [review]
Looking good Brett, I think this is good to merge! As we add in tests, this'll help increase confidence in new features/refactors.

**Pull request merged**